src/modules/orders/templates/email/email.templates.ts

```
@@ -0,0 +1,14 @@
+export function orderStatusTemplate(email: string, orderId: string, status: string) {
```

ESLint

[error] Replace `email:·string,·orderId:·string,·status:·string` with `⏎··email:·string,⏎··orderId:·string,⏎··status:·string,⏎`

(prettier/prettier)